### PR TITLE
fix bug in Dockerfile for python

### PR DESCRIPTION
**Pull request by Panlichen**
#### Summary

fix typo in Dockerfile to make it work with ` --grpc_python_out=` option.

sorry for other chaos in code.
...

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,12 +23,12 @@ RUN apk add --no-cache automake && \
     cd /protobuf && \
     ./autogen.sh && \
     ./configure --prefix=/usr --enable-static=no && \
-    make -j4 && \
-    make -j4 check && \
-    make -j4 install && \
-    make -j4 install DESTDIR=/out && \
+    make -j21 && \
```
Comment (yurishkuro) [inline Dockerfile]: why does this need to change?

**Review by yurishkuro — CHANGES_REQUESTED** [review]
Please undo changes to unrelated files

**Comment by Panlichen** [comment]
I work on a 20 core platform. Maybe it is better to automize this
configuration, but I do not know how to do it in Dockerfile.

Yuri Shkuro <notifications@github.com> 于 2020年11月13日周五 01:53写道：

**Comment by yurishkuro** [comment]
> I work on a 20 core platform. 

The build runs on Travis-ci

**Comment by Panlichen** [comment]
In the new commit, I rollback the -j change. I am not familiar with Travis-ci, does the default setting for it use a four-core platform?

**Comment by Panlichen** [comment]
Will this tiny fix be merged?

**Comment by yurishkuro** [comment]
Why is it even a bug? It seems consistent with the naming of all other plugins. What is the reproducer that demonstrates that this does not work?

**Pull request closed without merging**